src/main/java/DukeException.java

```diff
@@ -0,0 +1,71 @@
+public class DukeException{
+    private Ui ui = new Ui();
+
+    public DukeException(){
+    }
+
+    /**
+     * This method prints the Error Message for Incorrect Input.
+     */
+    public void IncorrectInputTodo(){
+        System.out.print(ui.line() + "☹ OOPS!!! The description of a todo cannot be empty.\nPlease insert Action\n" + ui.line());
+    }
+
+    /**
+     * This method prints the Error Message for Incorrect Input for a Deadline Task.
+     */
+    public void IncorrectInputDeadline(){
+        System.out.print(ui.line() + "☹ OOPS!!! The description of a Deadline cannot be empty.\nPlease insert Action\n" + ui.line());
```

Comment:
This line is relatively long. You might want to consider keeping this line under 120 characters.